snuba/datasets/schemas/join.py

```diff
@@ -75,28 +76,7 @@ def get_schemas(self) -> Mapping[str, Schema]:
 
 
 @dataclass(frozen=True)
-class SubJoinSource(JoinedSource):
```

Comment:
I think this had a reason to exist at the beginning, but after several rounds of refactoring I missed the fact it did not bring anything anymore.